ivy/container/container.py

```diff
@@ -93,15 +93,17 @@ def __pos__(self):
         return self
 
     def __neg__(self):
-        return self.map(lambda x, kc: -x)
+        return self.map(lambda x, kc: -x, map_sequences=True)
```

Comment:
this is fine, but I think we should generalize `self.map` to also handle `map_nests` argument right? Maybe let's do this is another PR though.

Reply:
Currently, `self.map` function can handle `map_nests` with a different variable name `map_sequences` . Do you mean to set `map_sequences` as True by default or what does generalizing `self.map` refer to?